src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_7001_8000.cpp

```diff
@@ -635,6 +634,21 @@ GUI_TEST_CLASS_DEFINITION(test_7367) {
     CHECK_SET_ERR(percentT >= model.percentT - diff && percentT <= model.percentT + diff, "Invalid percent of T: " + QString::number(percentT));
 }
 
+GUI_TEST_CLASS_DEFINITION(test_7368) {
+    // Generate a large sequence (>=100mb).
+    // Check that error dialog is shown when such a big sequence is exported as an alignment.
+    DNASequenceGeneratorDialogFillerModel model(sandBoxDir + "/test_7368.fa");
+    model.length = 100 * 1000 * 1000;
```

Comment:
`MAX_SAFE_ALIGNMENT_SIZE_TO_EXPORT` is a public variable, you can use it here

Reply:
Unfortunately one plugin can't depend on another plugin and the constant is declared in the dna_export plugin.

I think we need to create something like "UGENE Limits".h in U2Core. I will do it in a separate patch and will move this cosntant there.